api/bazel/envoy_http_archive.bzl

```diff
@@ -0,0 +1,33 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load(":repository_locations.bzl", "REPOSITORY_LOCATIONS")
+
+def envoy_http_archive(name, repository_locations = REPOSITORY_LOCATIONS, **kwargs):
+    # `existing_rule_keys` contains the names of repositories that have already
+    # been defined in the Bazel workspace. By skipping repos with existing keys,
+    # users can override dependency versions by using standard Bazel repository
+    # rules in their WORKSPACE files.
+    existing_rule_keys = native.existing_rules().keys()
+    if name in existing_rule_keys:
+        # This repository has already been defined, probably because the user
+        # wants to override the version. Do nothing.
+        return
+
+    loc_key = kwargs.pop("repository_key", name)
+    location = repository_locations[loc_key]
+
+    # Git tags are mutable. We want to depend on commit IDs instead. Give the
```

Comment:
I think git repositories are already disabled so we can remove this.

Reply:
@lizan where are they disabled?